sirepo/numpy.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+u"""Wrappers for numpy
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import numpy as np
+
+def ndarray_from_csv(path, skip_header, **kwargs):
+    def _read():
+        import csv
+        import re
+
+        with open(path) as f:
```

Comment:
We should either use py.path or `open(...,"rt")`.

Reply:
83cb6542